### PR TITLE
EOS-22259 bound durability type to sns or dix

### PR DESCRIPTION
**Pull request by zahidsk**

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: lr-cli/cortx_setup/commands/storageset/config/durability.py [hunk 1]
```diff
@@ -65,45 +64,47 @@ class DurabilityConfig(Command):
 
     def run(self, **kwargs):
         try:
-            node_id = local_minion_id()
             index = 'storage_durability_index'
             storage_set_name = kwargs['storage_set_name']
             durability_type = kwargs['durability_type']
             cluster_id = get_cluster_id()
-
+            if kwargs['durability_type'] not in ['sns', 'dix']:
```
Comment (sradhanand-pati) [inline lr-cli/cortx_setup/commands/storageset/config/durability.py:71]: add choices in args defined above..
argparser will handle it internally

File: lr-cli/cortx_setup/commands/storageset/config/durability.py [hunk 2]
```diff
@@ -65,45 +64,47 @@ class DurabilityConfig(Command):
...
             self.load_conf_store(
                 index,
                 f'json://{CONFSTORE_CLUSTER_FILE}'
             )
 
-            ss_name = Conf.get(index, f'cluster>{cluster_id}>storage_set[0]>name')
-
-            durability_dict = {durability_type: {}}
+            ss_name = Conf.get(index,
+                               f'cluster>{cluster_id}>storage_set[0]>name')
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/storageset/config/durability.py:80]: Try to accommodate multiple storage-set as well here...

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
The code changes does not export latest values to ConfStore (Please fix that issue)

**Comment by sradhanand-pati** [comment]
Closing it as it will be addressed in  - https://github.com/Seagate/cortx-prvsnr/pull/6017

**Pull request closed without merging**